### PR TITLE
Remove default loss function from generators

### PR DESCRIPTION
**Pull request by alexandrevicenzi**
If the loss function is set as a default argument it will be loaded even
if not used, which is not desired.

FeatureLoss is used only for training, but was being loaded also for
inference, using a little more memory than required due VGG16 features.

This will not cause any harm because training and inference usage always
set gen_loss argument.

### CODE REVIEW

**Comment by alexandrevicenzi** [comment]
@jantic based on you comment on #168 about changs on `loss.py` I tracked down where `FeatureLoss` was being used for inference and it was because `gen_loss` had it as a default value, in this case, being loaded even if not in use.

This shouldn't cause any harm to training notebooks.
